### PR TITLE
Integration tests snowflake

### PR DESCRIPTION
**Pull request by bjornj12**
Add snowflake integration tests

### CODE REVIEW

**Review by hugihlynsson — COMMENTED** [review]
Lovely looking SQL 😍

File: integration_tests/profiles.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 integration_tests:
-  target: bigquery
+  target: snowflake
```
Comment (logason) [inline integration_tests/profiles.yml:2]: How does the target work here, now when we have both bigquery and snowflake tests in the outputs? 🤔

File: macros/audit_event_volume.sql [hunk 2]
```diff
@@ -168,26 +168,17 @@ avarage as (
...
-
--- Only return event_name/source combinations that have spiked or dropped for at least 1 day.
--- Disregard all combinations that were iniside the norm for the whole time period to reduce noise.
-select 
-  * 
-from aggregate_by_day_asc
-where
-    (select signal FROM UNNEST(signals) AS signal where signal = 1 GROUP BY signal) = 1
-    OR (select signal from UNNEST(signals) AS signal where signal = -1 GROUP BY signal) = -1
+{{avo_audit.find_signals('aggregate_by_day_asc')}}
```
Comment (logason) [inline macros/audit_event_volume.sql:181]: Nice! So here we've modified the sql to make sure it can run in both BigQuery and Snowflake? 💥

**Review by logason — APPROVED** [review]
LGTM! Left couple of questions:

File: integration_tests/profiles.yml (hunk 1 above)
Comment (bjornj12) [reply integration_tests/profiles.yml:2]: target here is the default target.

which means by default we will upload to bigquery, unless running `dbt run --target snowflake`

File: macros/audit_event_volume.sql (hunk 2 above)
Comment (bjornj12) [reply macros/audit_event_volume.sql:181]: yep! UNNEST was not possible in snowflake so we use LATERAL FLATTEN instead 👌

**Comment by bjornj12** [comment]
@logason

**Pull request merged**